### DISCUSSION TITLE
PDP - Find out responsible policy for DENY/ALLOW

### DISCUSSION THREAD

**Discussion by bbodensieck** [cerbos/cerbos#1588] [Q&A] [answered]
Hello everyone,

is it possible to find out which policy was responsible (the strongest) for the relevant DENY/ALLOW in a check?

Thanks
Björn

**Comment by charithe** [comment, marked as answer]
Hi. If you add `includeMeta: true` to the request, the response will include a `meta` section that tells you which policy produced the decision. See number 17 in the request block and number 6 in the response block at https://docs.cerbos.dev/cerbos/latest/api/index.html#check-resources for an example.

**Reply by bbodensieck** [reply]
Awesome, thank you
